config/locales/bp/ru.new_customer.yml

```diff
@@ -7,6 +7,7 @@ ru:
       homsOrderDataCustomerStreet: Улица
       homsOrderDataCustomerHouse: Дом
       homsOrderDataCustomerEntrance: Подъезд
+      static: <div class="alert alert-success"><h4>Подключение доступно</h4> <small>Решение основано на автоматической проверке доступа в биллинге</small></div>
```

Comment:
This form has no such field, check actual business process, please
https://github.com/latera/camunda-ext/blob/b0647bc3ad79a3bfcc06ecb00c388f8b2eb95ef9/demo_processes/new_customer/src/main/resources/forms/plan.yml#L4

Reply:
done